src/main/java/com/google/api/codegen/go/GoContextCommon.java

```diff
@@ -34,4 +35,34 @@
     }
     return result;
   }
+
+  public Iterable<String> getCommentLinesWrap(String text) {
+    List<String> result = new ArrayList<>();
+    for (String line : Splitter.on(String.format("%n")).split(text)) {
+      for (String wrapped : wrapLine(line, 70)) {
+        result.add(line.isEmpty() ? "//" : "// " + wrapped);
+      }
+    }
+    return result;
+  }
+
+  private List<String> wrapLine(String line, int length) {
+    if (line.length() <= length) {
+      return Collections.<String>singletonList(line);
+    }
+    List<String> result = new ArrayList<>(line.length() / length + 1);
+    StringBuilder current = new StringBuilder();
+    for (String word : Splitter.onPattern("\\s").omitEmptyStrings().split(line)) {
+      current.append(" ");
+      current.append(word);
+      if (current.length() >= length) {
```

Comment:
instead of checking that the new word begins within length, you could check the current.length() + word.length() <= length. It might make things more intuitive.